### PR TITLE
[ADD][11.0] Adding module lira_workflow_transitions_by_task_type

### PR DESCRIPTION
**Pull request by pnajman-modoolar**
Hello,

Module ``lira_workflow_transitions_by_task_type`` integrates modules [lira](https://github.com/OCA/project-agile/pull/10) and [project_agile_workflow_transitions_by_task_type](https://github.com/OCA/project-agile/pull/8).

Kind Regards,
Petar

### CODE REVIEW

File: lira_workflow_transitions_by_task_type/README.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline lira_workflow_transitions_by_task_type/README.rst:36]: see https://github.com/OCA/web/pull/881#pullrequestreview-99094143

File: lira_workflow_transitions_by_task_type/README.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
+.. image:: https://www.gnu.org/graphics/lgplv3-147x51.png
+   :target: https://www.gnu.org/licenses/lgpl-3.0.en.html
+   :alt: License: LGPL-v3
+
+======================================
+Lira Workflow Transitions By Task Type
+======================================
+
+This module integrates module ``project_workflow_transitions_by_task_type`` with module ``lira``.
```
Comment (elicoidal) [inline lira_workflow_transitions_by_task_type/README.rst:9]: With time a more complete README with proper links to Lira project is welcome

**Review by elicoidal — CHANGES_REQUESTED** [review]

File: lira_workflow_transitions_by_task_type/README.rst (hunk 1 above)
Comment (pnajman-modoolar) [reply lira_workflow_transitions_by_task_type/README.rst:36]: Fixed

File: lira_workflow_transitions_by_task_type/README.rst (hunk 2 above)
Comment (pnajman-modoolar) [reply lira_workflow_transitions_by_task_type/README.rst:9]: Sure ....

**Pull request closed without merging**